bsdlib/doc/gnss_extension.rst

```diff
@@ -96,31 +96,34 @@ The fix interval parameter controls the time between GNSS receiver-starts.
 It also controls the stopping of the GNSS receiver after a valid PVT
 estimate.
 The fix interval parameter determines the mode of navigation.
-There are three different navigation modes available: single-fix,
-continuous, and periodic.
+There are three different navigation modes available: single-fix, continuous, and periodic.
 
-Single-fix navigation mode is engaged by setting the fix interval to 0.
+Single-fix navigation mode is engaged by setting the fix interval to zero.
```

Comment:
I think it would be good to point out in the docs that the retry value is not used in single-fix mode. It will not produce a timeout similar to that of periodic mode. This has been a recurring source of confusion on DevZone.